configuration/ui_testing.md

```diff
@@ -18,6 +18,9 @@ The next step is to start the Go agent as an application.
 1.  Click on Start → All Programs → Go Agent → Run Go Agent.
 2.  To get the Go agent to start every time you log in, copy the Run Go Agent shortcut to the Startup folder of your start menu.
 
+
+_**Please note**_ : If the go-agent as an application is run by a non-administrative user, then the non-administrative user needs to have these required permissions to the go-agent folder, i.e. modify, read and execute, list folder contents and read permissions.
```

Comment:
Local system administrator is different from administrator. So, to specify non-administrator is misleading in my opinion. Also, this information must be present in the documentation that talks about installation. And must be there for both server and agent.